addons/account/views/account_tax_views.xml

```diff
@@ -198,9 +198,22 @@
                 <tree string="Account Tax Group" editable="bottom" create="false">
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
-                    <field name="property_tax_payable_account_id"/>
-                    <field name="property_tax_receivable_account_id"/>
-                    <field name="property_advance_tax_payment_account_id"/>
+
+                    <!--
+                        'force_company' context key is used so that the RedirectWarning
+                        potentially raised during tax closing doesn't allow the user choosing
+                        accounts from another company in case the multicompany selector
+                        currently grants access to multiple companies.
+                    -->
+                    <field name="property_tax_payable_account_id"
+                        domain="[('company_id', '=', context['force_company'])] if context.get('force_company') else []"
+                    />
+                    <field name="property_tax_receivable_account_id"
+                        domain="[('company_id', '=', context['force_company'])] if context.get('force_company') else []"
+                    />
+                    <field name="property_advance_tax_payment_account_id"
+                        domain="[('company_id', '=', context['force_company'])] if context.get('force_company') else []"
```

Comment:
I'm pretty sure `force_company` is no longer used in latest version. You should use `allowed_company_id`/`allowed_company_ids` instead I think

Reply:
I define it here https://github.com/odoo/enterprise/pull/18234/files#diff-3aeae56c03ab4143f75a516c7cf9066f62d00eb6e226ac123bbeba108f90f3ddR351

allowed_company_id doesn't exist. I could rely on allowed_company_ids[0], but apparently our js Python parser doesn't support array indexing in domains (I quickly checked py.js after getting the error). So I guess this is the best we can do :/